src/contexts/player.js

```diff
@@ -0,0 +1,18 @@
+const {createContext, useContext, useState} = require('react')
+
+const PlayerContext = createContext()
+
+function PlayerProvider(props) {
+	const [channel, setChannel] = useState()
+	const [track, setTrack] = useState()
+	const value = {channel, setChannel, track, setTrack}
+	return <PlayerContext.Provider value={value}>{props.children}</PlayerContext.Provider>
```

Comment:
it took me time to understand what was happening here.
Is the second block after the return? Not it is the closing block, "ah this must be returning the two lines). Why not 
```
return (
  <>
  </>
)
```
the 2 extra lines?

Reply:
my bad, was looking in github diff interface too zoomed in and appeared on two lines